app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -91,7 +92,7 @@ def pii_from_doc
             pii[idp_key] = true_id_product[:IDAUTH_FIELD_DATA][true_id_key]
           end
 
-          pii[:state_id_type] = 'drivers_license'
+          pii[:state_id_type] = DocAuth::Response::ID_TYPE_SLUGS[pii[:state_id_type]]
```

Comment:
Should we update these assertions as well, so we have some coverage for LN?

https://github.com/18F/identity-idp/blob/e5b72af77401c00508cefabad56072ca054923e8/spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb#L65-L71

https://github.com/18F/identity-idp/blob/e5b72af77401c00508cefabad56072ca054923e8/spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb#L96-L102